src/lib/util/copy/__tests__/copy.test.js

```diff
@@ -0,0 +1,36 @@
+import copy from '../copy';
+
+describe('copy', () => {
+    it('copies the provided text to the clipboard', () => {
+        const writeText = jest.fn();
+
+        global.navigator.clipboard = {
```

Comment:
These tests that override `global.navigator.clipboard`... I think we should make sure to set this back to the original value after the test in order to ensure consistent behavior in any other tests that might use `writeText`.  Or am I missing something that would make this not a problem?

Reply:
Thanks! I wasn't sure those changes carried between tests. Either way, I've added a reset after each that should suffice. Let me know if you see any further issues.

Reply:
I wasn't 100% sure if the changes carried either, but adding the reset does seem safer.  Thanks!